### PR TITLE
DCS-2232 code refactor

### PR DESCRIPTION
**Pull request by GurnankCheema**
create config var for female prisons and use in view templates

### CODE REVIEW

File: server/routes/homeController.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import type { RequestHandler } from 'express'
+
+export default class HomeController {
```
Comment (GurnankCheema) [inline server/routes/homeController.ts:3]: new controller so could create suitable tests

File: server/config.ts [hunk 2]
```diff
@@ -107,6 +107,7 @@ export default {
     tagManagerEnvironment: get('TAG_MANAGER_ENVIRONMENT', ''),
   },
   enabledPrisons: get('ENABLED_PRISONS', '', requiredInProduction).split(','),
+  femalePrisons: get('FEMALE_PRISONS', '', requiredInProduction).split(','),
   confirmEnabled: get('CONFIRM_ENABLED', 'false', requiredInProduction) === 'true',
```
Comment (GurnankCheema) [inline server/config.ts:111]: set female prisons as config var

File: server/config.ts [hunk 3]
```diff
@@ -107,6 +107,7 @@ export default {
     tagManagerEnvironment: get('TAG_MANAGER_ENVIRONMENT', ''),
   },
   enabledPrisons: get('ENABLED_PRISONS', '', requiredInProduction).split(','),
+  femalePrisons: get('FEMALE_PRISONS', '', requiredInProduction).split(','),
```
Comment (GurnankCheema) [inline server/config.ts:110]: set female prisons as a config var

File: server/routes/index.ts [hunk 4]
```diff
@@ -10,9 +11,9 @@ import feedbackRoutes from './feedback'
 
 export default function routes(services: Services): Router {
   const prisonerController = new PrisonerController(services.expectedArrivalsService)
-
+  const homeController = new HomeController()
   return Routes.forAnyRole()
-    .get('/', (_, res) => res.render('pages/home'))
+    .get('/', homeController.view())
```
Comment (GurnankCheema) [inline server/routes/index.ts:16]: view now captured in a controller

**Review by davidthomas22 — APPROVED** [review]
LGTM

**Pull request merged**